pandas/tests/interchange/test_impl.py

```diff
@@ -9,12 +9,12 @@
 
 import pandas as pd
 import pandas._testing as tm
+from pandas.core.interchange._from_dataframe import from_dataframe
 from pandas.core.interchange.column import PandasColumn
 from pandas.core.interchange.dataframe_protocol import (
     ColumnNullType,
     DtypeKind,
 )
-from pandas.core.interchange.from_dataframe import from_dataframe
```

Comment:
could you address my comment about importing `from pandas.core.interchange import from_dataframe` directly please?